src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1135,14 +1139,17 @@ private function registerDebugConfiguration(array $config, ContainerBuilder $con
     {
         $loader->load('debug_prod.php');
 
+        $debug = $container->getParameter('kernel.debug');
+
         if (class_exists(Stopwatch::class)) {
```

Comment:
since the service is optionally defined, we should ensure that Application can deal with a missing stopwatch (aka display something and skip)

we could also display something when --profile is used by debug is false BTW

Reply:
Done in last commit.